docs/src/pinn/neural_adapter.md

```diff
@@ -0,0 +1,252 @@
+## Neural Adapter
+
+Using the example of 2D Poisson equation, it is shown how, using method neural_adapter, to retrain the prediction of one neural network to another.
```

Comment:
This needs more of an introduction. What is transfer learning? Why would you want to use it? What are the pros? The cons?